<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbPorts">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="advanced">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Advanced</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Baudrate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbBaudrate">
          <property name="currentIndex">
           <number>12</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAlphabetically</enum>
          </property>
          <item>
           <property name="text">
            <string>50</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>75</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>110</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>134</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>150</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>230400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>460800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>576000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>921600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1000000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1152000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1500000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2000000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2500000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3000000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3500000</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4000000</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>ByteSize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbByteSize">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>5-Bits</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6-Bits</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7-Bits</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8-Bits</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Parity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbParity">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Even</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Odd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mark</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Space</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Stopbits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbStopbits">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Flow Control</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbFlowControl">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hardware</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Software</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Unknown</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Abort|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
